### PR TITLE
allow to specify database name

### PR DESCRIPTION
**Pull request by trhura**
Firestore now supports multiple database name and this PR add support for modifying the database name (from the `(default)`).

### CODE REVIEW

File: client.go [hunk 1]
```diff
@@ -18,31 +19,23 @@ func createClient(credentials string) (*firestore.Client, error) {
...
-		firebaseApp, err = firebase.NewApp(context.Background(), config)
-		if err != nil {
-			return nil, err
-		}
-		return firebaseApp.Firestore(context.Background())
+		return firestore.NewClient(context.Background(), projectId)
+	}
+
+	if database == "" {
+		database = "(default)" // use default database if not specified
```
Comment (sgarciac) [inline client.go:30]:
```suggestion
		database = firestore.DefaultDatabaseID
```

**Comment by sgarciac** [comment]
It looks good to me. I'm just wondering if the new huge go.sum is normal 😅

**Comment by sgarciac** [comment]
@trhura hello, could you please confirm that the go.sum is correct (see my previous comment)? Thanks!

**Comment by trhura** [comment]
Not really sure why it is huge - all I did was update `cloud.google.com/go/firestore`. Maybe different go toolchain and library versions triggered this.  

Anyway, I have deleted and regenerated `go.mod` and `go.sum`.

**Review by sgarciac — APPROVED** [review]

**Pull request merged**